### PR TITLE
Create fr_FR.json

### PR DESCRIPTION
**Pull request by Luna664**

### CODE REVIEW

**Comment by Fayti1703** [comment]
Could you convert the file into the same format used by `en_US.json` and `de_DE.json`? It's currently still in the old, legacy format, which won't work on this branch.

(no need to create a new PR, you can just push additional commits onto your fork's branch and they should show up here)

**Comment by Luna664** [comment]
Sorry I had checked the old ENGLISH.json file instead of the En_US.json file.

The file is normally converted as it should be now

File: Ktisis/Locale/i18n/fr_FR.json [hunk 1]
```diff
@@ -1,280 +1,407 @@
...
+						"release": "Relâchez",
+						"__comment": "Cette clé n'est (espérons-le) jamais utilisée.",
+						"invalid": "N/A"
+					},
+					"GlobalModifierKey": "Modificateur global",
+					"SwitchToTranslate": "Appuyer pour traduire",
+					"SwitchToRotate": "Passer à la rotation",
+					"SwitchToScale": "Passer à l'échelle",
+					"SwitchToUniversal": "Passer à l'universel",
+					"ToggleLocalWorld": "Basculer local/monde",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:131]: Maybe rajouter `mode` ? This setting should control how the rotation works

File: Ktisis/Locale/i18n/fr_FR.json [hunk 2]
```diff
@@ -1,280 +1,407 @@
...
+		"j_f_lip_r": "Lèvre Droite",
+		"j_f_mayu_l": "Sourcil Gauche",
+		"j_f_mayu_r": "Sourcil Droit",
+		"j_f_memoto": "Arcade Centrale",
+		"j_f_miken_l": "Arcade Gauche",
+		"j_f_miken_r": "Arcade Droite",
+		"j_f_ulip_a": "Lèvre supèrieur Droite",
+		"j_f_umab_l": "Paupière Gauche",
+		"j_f_umab_r": "Paupière Droite",
+		"j_f_dlip_a": "Lèvre inferieure A",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:349]:
```json
"j_f_dlip_a": "Lèvre inférieure A"
```

File: Ktisis/Locale/i18n/fr_FR.json [hunk 3]
```diff
@@ -1,280 +1,407 @@
...
+		"j_ex_met_va": "Visière",
+		"mh_n_hara": "Main Droite",
+		"oh_n_hara": "Main Gauche",
+		"j_half_l": "Côté Gauche",
+		"j_half_r": "Côté Droit",
+	},
+
+	"boneCategory": {
+		"other": "Autre",
+		"body": "Corp",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:386]:
```json
"body": "Corps",
```

File: Ktisis/Locale/i18n/fr_FR.json [hunk 4]
```diff
@@ -1,280 +1,407 @@
...
+		"head": "Tête",
+		"hair": "Cheveux",
+		"clothes": "Vêtements",
+		"weapons": "Armes",
+		"right hand": "Main Droite",
+		"left hand": "Main Gauche",
+		"feet": "Pied",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:393]:
```json
"feet": "Pieds",
```

File: Ktisis/Locale/i18n/fr_FR.json [hunk 5]
```diff
@@ -1,280 +1,407 @@
...
+		"hands": "Mains",
+		"tail": "Queue",
+		"ears": "Oreilles",
+		"ivcs left hand": "IVCS Main Gauche",
+		"ivcs right hand": "IVCS Main Droite",
+		"ivcs left foot": "IVCS Pied Gauche",
+		"ivcs right foot": "IVCS Pied Droit",
+		"ivcs body": "IVCS Corp",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:401]:
```json
"ivcs body": "IVCS Corps"
```

**Review by therealedited — CHANGES_REQUESTED** [review]
There are some grammatical errors that need to be fixed and some context related to the tool that needs to be added. Feel free to argue about any requested changes.

File: Ktisis/Locale/i18n/fr_FR.json [hunk 6]
```diff
@@ -1,280 +1,407 @@
...
+			},
+			"transformTable": {
+				"title": "Table des transformations",
+				"digitPrecision": "Précision des chiffres",
+				"basePositionSpeed": "Vitesse de base (Position)",
+				"baseRotationSpeed": "Vitesse de base (Rotation)",
+				"baseScaleSpeed": "Vitesse de base (Mise à l'échelle",
+				"ctrlSpeedMult": "Ctrl Multiplicateur de vitesse",
+				"shiftSpeedMult": "Maj Multiplicateur de vitesse",
+				"showSpeedMultiplierInputs": "Afficher les entrées du multiplicateur de vitesse"
```
[the remark this answers is not in the document]
Comment (Luna664) [reply Ktisis/Locale/i18n/fr_FR.json]: I didn't manage to translate this well, I'll talk about it around me to help me find a better way to say it

File: Ktisis/Locale/i18n/fr_FR.json [hunk 7]
```diff
@@ -1,280 +1,407 @@
...
+					"GlobalModifierKey": "Modificateur global",
+					"SwitchToTranslate": "Appuyer pour traduire",
+					"SwitchToRotate": "Passer à la rotation",
+					"SwitchToScale": "Passer à l'échelle",
+					"SwitchToUniversal": "Passer à l'universel",
+					"ToggleLocalWorld": "Basculer local/monde",
+					"HoldToHideSkeleton": "Maintenir pour cacher le squelette",
+					"ClearCategoryVisibilityOverload": "Effacer la surcharge de visibilité des catégories",
+					"HoldAllCategoryVisibilityOverload": "Maintenir la surcharge de visibilité de toutes les catégories",
+					"CircleThroughSiblingLinkModes": "Circle through sibling link modes"
```
[the remark this answers is not in the document]
Comment (Luna664) [reply Ktisis/Locale/i18n/fr_FR.json]: I didn't manage to translate this sentence too, I tried to do my best, but the translation might not fit

File: Ktisis/Locale/i18n/fr_FR.json [hunk 8]
```diff
@@ -1,280 +1,407 @@
...
+		"references": {
+			"title": "Références",
+			"explanation": "Superposez des images de référence sur votre jeu, pour vous aider à prendre la bonne pose.",
+			"imageAlpha": "Opacité",
+			"hideWindowDecoration": "Cacher la décoration des fenêtres",
+			"images": {
+				"title": "Images de référence",
+				"dialog": {
+					"title": "Ajouter un fichier de référence",
+					"filter": "Fichiers d'images"
```
[the remark this answers is not in the document]
Comment (Luna664) [reply Ktisis/Locale/i18n/fr_FR.json]: I think "Image files" is more correct, but this is my personal opinion, both can be used

File: Ktisis/Locale/i18n/fr_FR.json [hunk 9]
```diff
@@ -55,7 +55,7 @@
 				"drawLines": "Dessiner des lignes sur le squelette",
 				"__comment": "NOTE : Ceux-la devraient refléter plus clairement la fonctionnalité réelle (dessiner des lignes/points lorsque le *gizmo est actif*)."",
 				"drawLineWhenSelecting": "Tracer des lignes lors de la sélection",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:57]:
```json
"drawLineWhenSelecting": "Afficher une ligne lors d'une sélection"
```
sound more in-context to me but I'm not exactly sure where the english text is displayed. Same for the one below `Afficher un point lors d'une selection`

File: Ktisis/Locale/i18n/fr_FR.json [hunk 10]
```diff
@@ -1,280 +1,407 @@
...
+					"generic": {
+						"__comment": "Ils sont utilisés pour les raccourcis clavier des catégories. (Ils seront probablement remplacés par un système de caractères génériques à l'avenir).",
+						"toggle": "Basculer",
+						"hold": "Maintenir",
+						"release": "Relâchez",
+						"__comment": "Cette clé n'est (espérons-le) jamais utilisée.",
+						"invalid": "N/A"
+					},
+					"GlobalModifierKey": "Modificateur global",
+					"SwitchToTranslate": "Appuyer pour traduire",
```
[the remark this answers is not in the document]
Comment (Luna664) [reply Ktisis/Locale/i18n/fr_FR.json:127]: I also failed to translate this correctly, I will talk about it to find a correct translation

File: Ktisis/Locale/i18n/fr_FR.json [hunk 11]
```diff
@@ -105,8 +105,8 @@
 				"enable": "Activer",
 
 				"assignByKeyPress": {
-					"title": "Presser les touches",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json]: I still don't think it's correct. This text displays in the Input tab and basically shows the user what is the key (or shortcut) that they're currently pressing. `Touches enfoncées` ?

File: Ktisis/Locale/i18n/fr_FR.json [hunk 12]
```diff
@@ -291,16 +291,16 @@
 		"j_te_r": "Main Droite",
 		"n_buki_tate_l": "Bouclier Gauche",
 		"n_buki_tate_r": "Bouclier Droit",
-		"n_ear_a_l": "Boucle d'oreille A Gauche",
-		"n_ear_a_r": "Boucle d'oreille A Droite",
+		"n_ear_a_l": "Boucles d'oreille A Gauche",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:294]: `Oreille` takes a "s" if you decide to use `Boucles`

File: Ktisis/Locale/i18n/fr_FR.json [hunk 13]
```diff
@@ -343,12 +343,12 @@
 		"j_f_memoto": "Arcade Centrale",
 		"j_f_miken_l": "Arcade Gauche",
 		"j_f_miken_r": "Arcade Droite",
-		"j_f_ulip_a": "Lèvre supèrieur Droite",
+		"j_f_ulip_a": "Lèvre supérieur Droite",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:346]: `Lèvre` is feminine. So it's `supérieure`

File: Ktisis/Locale/i18n/fr_FR.json [hunk 14]
```diff
@@ -29,21 +29,21 @@
 				"openMethod": {
 					"fieldLabel": "Ouvrir Ktisis",
 					"Manually": "manuellement via `/ktisis`",
-					"OnPluginLoad": "lorsque le plugin se charge",
+					"OnPluginLoad": "lorsque le plugin se lance",
 					"OnEnterGpose": "lors de la pose en groupe"
 				},
 				"hideCharName": "Cacher le nom du personnage",
-				"censorNSFW": "Censure NSFW"
+				"censorNSFW": "Cacher le contenue explicite"
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:36]: `contenu` and not `contenue`

**Review by therealedited — CHANGES_REQUESTED** [review]

File: Ktisis/Locale/i18n/fr_FR.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (Luna664) [reply Ktisis/Locale/i18n/fr_FR.json]: A translation was finally found by a friend

File: Ktisis/Locale/i18n/fr_FR.json [hunk 15]
```diff
@@ -0,0 +1,407 @@
+{
+	"$meta": {
+		"displayName": "French",
+		"selfName": "French",
+		"__comment": "Utilisation de l'élément null pour signaler 'et al' dans le tableau ci-dessous. Ne faites pas cela pour votre propre localisation.",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:5]: By the way, there was no need to translate `__comment` stuff as this is used for translators and will never be displayed in game.

File: Ktisis/Locale/i18n/fr_FR.json [hunk 16]
```diff
@@ -0,0 +1,407 @@
+{
+	"$meta": {
+		"displayName": "French",
+		"selfName": "French",
```
Comment (therealedited) [inline Ktisis/Locale/i18n/fr_FR.json:4]: `selfName` should be `Français`

**Review by therealedited — CHANGES_REQUESTED** [review]

File: Ktisis/Locale/i18n/fr_FR.json [hunk 17]
```diff
@@ -0,0 +1,407 @@
...
+			"glamourDresser": {
+				"__comment": "Ceux-ci sont actuellement un peu désordonnés... Besoin d'un système de placement pour corriger cela.",
+				"memoryCount": "Planche mirage en mémoire : ",
+				"validCount": {
+					"pre": "Trouvé ",
+					"post": "Planche mirage valides"
+				},
+
+				"sync": {
+					"tooltip": "Rafraîchit la mémoire de la planche mirage pour les recherches de sets. \NCette mémoire est conservée après un redémarrage.\N Conditions requises:\NL'une de ces fenêtres doit être ouverte : \"Création de planche mirage\" (par la Coiffeuse mirage) ou \"Sélection de planche mirage\" (avec la compétence Planche mirage)."
```
Comment (Fayti1703) [inline Ktisis/Locale/i18n/fr_FR.json:173]: Lower case `\n` here, please. `\N` is not a valid JSON escape sequence.

File: Ktisis/Locale/i18n/fr_FR.json [hunk 18]
```diff
@@ -0,0 +1,407 @@
...
+			}
+		},
+
+		"overlay": {
+			"title": "Superposition",
+
+			"skeleton": {
+				"header":"Lignes et points du squelette",
+				"drawLines": "Dessiner des lignes sur le squelette",
+				"__comment": "NOTE : Ceux-la devraient refléter plus clairement la fonctionnalité réelle (dessiner des lignes/points lorsque le *gizmo est actif*)."",
```
Comment (Fayti1703) [inline Ktisis/Locale/i18n/fr_FR.json:56]: JSON syntax error here.
```suggestion
				"__comment": "NOTE : Ceux-la devraient refléter plus clairement la fonctionnalité réelle (dessiner des lignes/points lorsque le *gizmo est actif*).",
```

File: Ktisis/Locale/i18n/fr_FR.json [hunk 19]
```diff
@@ -0,0 +1,407 @@
+{
+	"$meta": {
+		"displayName": "French",
+		"selfName": "Français",
+		"__comment": "Using null element to signal 'et al' in the array below. Don't do this for your own localization.",
+		"maintainers": [
+			"@Luna664",
+			null
```
Comment (Fayti1703) [inline Ktisis/Locale/i18n/fr_FR.json:8]: As the comment above states, "Don't [use a null element] for your own localization".

**Review by Fayti1703 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**